lib/classes/Swift/Plugins/AntiFloodPlugin.php

```diff
@@ -131,7 +131,9 @@ public function sleep($seconds)
         if (isset($this->sleeper)) {
             $this->sleeper->sleep($seconds);
         } else {
-            sleep($seconds);
+            do {
+                $seconds = sleep($seconds);
```

Comment:
The PHP docs mention that on Windows, it always returns 192.
Do you have any use cases in mind that makes this change required?

Reply:
In case the SwiftMailer instanced as a server worker that handle the sending queue and the sending is being throttled, the SIGUSR1 call interrupt the sleeping, that perform the excess with sending.

As far as I know, Windows don't use POSIX signals, so we can implement some condition handling that case. Anyhow, the algo of default sleep becaming complex. Maybe it make sense to implement a Default Sleeper instead.